src/app/shared/directives/autofocus/directive.ts

```diff
@@ -26,16 +26,21 @@ export class AutofocusDirective implements AfterViewInit, OnDestroy {
 
   constructor(private readonly _el: ElementRef) {}
 
+  focus() {
```

Comment:
nit: Place lifecycle hook methods (in this case `ngAfterViewInit`) above the rest.

Reply:
Also, this method should be private as it will not be directly used inside the template.